js/demo/lazy-loading.js

```diff
@@ -74,7 +74,6 @@ var EffecktLazyLoading = {
       scrolled = $('.effeckt-page-active').scrollTop(),
       viewed = scrolled + this.viewportHeight,
       elTop = $(el).offset().top,
-      elBottom = elTop + elHeight,
```

Comment:
Same thing as above. This is IMHO worse at it's more than just a code style tweak.